mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -101,3 +114,69 @@ func ignitionFailure(c cluster.TestCluster) error {
 		return nil
 	}
 }
+
+func bootfsFailure(c cluster.TestCluster) error {
+	// We can't create files in / due to the immutable bit OSTree creates, so
+	// this is a convenient way to test Ignition failure.
+	failConfig, err := conf.EmptyIgnition().Render(conf.FailWarnings)
+	if err != nil {
+		return errors.Wrapf(err, "creating empty config")
+	}
+	cmd := exec.Command("/bin/bash", "-c", fmt.Sprintf(`set -euo pipefail;
```

Comment:
We should create this file in a directory that we know will get cleaned up. `builder`'s `tempdir` is a good candidate. We can add a new `GetTempdir()` method to get its path and then create this file in there.

Reply:
Testing it here:
https://github.com/coreos/coreos-assembler/pull/3647/commits/9376cceb2930966fff0cade45d3292fb669b88c7#diff-3ad99cf50017bf15cd0eae37f7571c6e217480c1c2b1748266bc7eb417cb0aacR123-R130